cmd/main.go

```diff
@@ -103,6 +104,10 @@ func main() {
 		PodResourceType string `envconfig:"POD_RESOURCE_TYPE" default:"google.com/tpu"`
 
 		Concurrency int `envconfig:"CONCURRENCY" default:"3"`
+
+		StaticNodepoolCreateTimeout time.Duration `envconfig:"STATIC_NODEPOOL_CREATE_TIMEOUT" default:"10m"`
+
+		ManagerSyncPeriod time.Duration `envconfig:"MANAGER_SYNC_PERIOD" default:"10m"`
```

Comment:
This probably should not happen on a manager-level basis, as the controller watches Pods and this would be expensive at large scale. Check if it is an option at the reconciler-level.